CODEOWNERS

```diff
@@ -1 +1 @@
-* @sebapawlak @wyrzyk @dagguh @mcmurphy2000
+* @atlassian/jpt
```

Comment:
I thought Atlassian is going to delete all the teams 🤷‍♂.

Reply:
I'm [pushing back](https://hello.atlassian.net/wiki/spaces/OSP/blog/2020/02/10/639043850/Re-+Introducing...+Open+Source+at+Atlassian?focusedCommentId=647724274#comment-647724274) on this